### PR TITLE
Adds crutches

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

Adds crutches, a direct medical-grade upgrade to canes for those with missing or damaged legs. They remove slowdown from missing/damaged legs better than canes when wielded. They can be found in medical closets or made at the protolathe. People missing a leg from roundstart will still get a cane, medical and science can provide some if the person really wants one.

EDIT: Since mapping freeze is over, I've added 3 crutches to each map, previously they were in lockers, but each map can have an inconsistent amount of lockers.

## Why It's Good For The Game
More flavor and options for those with disabilities is cool. (And more medical content)

## Images of changes

image

## Testing

Cut off my legs, got crutches, then it wasn't so bad. Started up an ss13 server and did the same thing (I also tested getting them from the lockers/protolathe, but thats not as funny)

## Changelog
:cl:
add: Medical crutches, to help people with missing/damaged legs.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/misc_items.dm [hunk 1]
```diff
@@ -33,8 +33,33 @@
...
 	attack_verb = list("bludgeoned", "whacked", "disciplined", "thrashed", "Vaudevilled")
 
-/obj/item/cane/is_crutch()
-	return 1
+/obj/item/cane/get_crutch_efficency()
+	return 2
+
+/obj/item/twohanded/crutches // someone move these out of lockers into the map once the freeze is over :) (so we can get a consistent amount for each map)
+	name = "crutches"
+	desc = "A medical device to help those walk who have injured or missing legs."
```
Comment (Henri215) [inline code/game/objects/items/weapons/misc_items.dm:41]:
```suggestion
	desc = "A medical device to help those who have injured or missing legs to walk."
```
Like this is better?

File: code/modules/research/designs/medical_designs.dm [hunk 2]
```diff
@@ -72,6 +72,16 @@
 	materials = list(MAT_METAL = 1000, MAT_SILVER = 500, MAT_GLASS = 500, MAT_DIAMOND = 200)
 	category = list("Medical")
 
+/datum/design/crutches
+	name = "Crutches"
+	desc = "A pair of crutches to help those walk who have injured or missing legs."
```
Comment (Henri215) [inline code/modules/research/designs/medical_designs.dm:77]:
```suggestion
	desc = "A pair of crutches to help those who have injured or missing legs to walk."
```

**Comment by Probablyjay19** [comment]
Can I use it like an axe? 👀

**Comment by McRamon** [comment]
Inhands are not good enough, try to improve them using the example i sent

**Comment by Contrabang** [comment]
> Inhands are not good enough, try to improve them using the example i sent

In hands updated

**Comment by Contrabang** [comment]
> Can I use it like an axe? 👀

Nope, if you want a crutch for combat, use a chainsaw :)

File: code/game/objects/items/weapons/misc_items.dm [hunk 3]
```diff
@@ -33,8 +34,33 @@
 	materials = list(MAT_METAL=50)
 	attack_verb = list("bludgeoned", "whacked", "disciplined", "thrashed", "Vaudevilled")
 
-/obj/item/cane/is_crutch()
-	return 1
+/obj/item/cane/get_crutch_efficency()
```
Comment (Henri215) [inline code/game/objects/items/weapons/misc_items.dm:37]: efficiency*

File: code/game/objects/items/weapons/misc_items.dm [hunk 4]
```diff
@@ -33,8 +34,35 @@
 	materials = list(MAT_METAL=50)
 	attack_verb = list("bludgeoned", "whacked", "disciplined", "thrashed", "Vaudevilled")
 
-/obj/item/cane/is_crutch()
-	return 1
+/obj/item/cane/get_crutch_efficiency()
+	return 2
+
+/obj/item/crutches // someone move these out of lockers into the map once the freeze is over :) (so we can get a consistent amount for each map)
```
Comment (Henri215) [inline code/game/objects/items/weapons/misc_items.dm:40]: Map freeze is over :)

**Review by Henri215 — APPROVED** [review]
LGTM (Lets Gamble, Try Mexico)

**Review by Git-GoR — CHANGES_REQUESTED** [review]
Mapping changes look good, apart from Cyberiad where design does not align with floor decorations. I would suggest something like this (cleaner does not deserve a rack only for one item tbh)
imagen

**Comment by Contrabang** [comment]
> Mapping changes look good, apart from Cyberiad where design does not align with floor decorations. I would suggest something like this (cleaner does not deserve a rack only for one item tbh) imagen

Changes applied

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**